.github/workflows/govulncheck_scan.yaml

```
@@ -0,0 +1,47 @@
+name: Govulncheck Security Scan
+
+on:
+  schedule:
+    - cron: '0 13 * * 1'
+  workflow_dispatch:
+
+permissions:
+  contents: read
+  security-events: write
+
+jobs:
+  govulncheck:
+    runs-on: lab
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+
+      - name: Setup Go
+        uses: actions/setup-go@v5
+        with:
+          go-version: stable
+          cache: true
+
+      - name: Install govulncheck
+        run: go install golang.org/x/vuln/cmd/govulncheck@latest
+
+      - name: Run govulncheck
+        run: govulncheck -show verbose ./...
+
+      - name: Generate SARIF file
+        if: success() || failure()  # Run even if previous step fails
```

```suggestion
        if: always()  # Run even if previous step fails
```

Should be the same (I think??) and is shorter?

---

Good point. Addressed